src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -626,6 +652,18 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
         });
     }
 
+    protected updateColumnVisibility(id: string, visible: boolean)
+    {
+        if (this.store.columnVisibility[id] !== undefined) {
+            this.store.updateColumnVisibility(id, visible);
```

Comment:
is it necessary to update both internal visibility state AND also call the onColumnVisiblilityUpdated callback, which presumably would update the external state of column visibility?

Reply:
`onColumnVisiblilityUpdated` reports back the current internal state to the parent component. But actually, I ended up not using it for this PR. I was thinking it might be useful for the patient view page, but we may remove it from this PR. I can add it back with the next PR (related to https://github.com/cBioPortal/cbioportal/issues/4026) if I need it.

Reply:
removed `onColumnVisiblilityUpdated` for now...